### PR TITLE
DES-9877: Fix screen snipping on secondary monitor

### PR DESCRIPTION
**Pull request by rbarros-daitan**

### CODE REVIEW

File: minuet/Paragon.Plugins.ScreenCapture/SnippingTool.cs [hunk 1]
```diff
@@ -40,7 +40,7 @@ public SnippingTool(Image screenShot, Rectangle position)
             base.Top = position.Top;
             base.Size = position.Size;
             base.DoubleBuffered = true;
-
+            StartPosition = FormStartPosition.Manual;
```
Comment (lneir) [inline minuet/Paragon.Plugins.ScreenCapture/SnippingTool.cs:43]: don't see this being used anywhere?

File: minuet/Paragon.Plugins.ScreenCapture/SnippingWindow.xaml.cs [hunk 2]
```diff
@@ -64,14 +66,17 @@ private void OnLoaded(object sender, RoutedEventArgs e)
             WindowState = WindowState.Normal;
 
             var rect = result.SelectedRectangle;
-
             // reposition the window so there's a neat effect of showing 
             // the screenshot edit window in place of the selected region
-            var top = rect.Top - 80;
-            var left = rect.Left - 80;
+            
+            var top = rect.Top - 80 + SystemInformation.VirtualScreen.Top;
```
Comment (lneir) [inline minuet/Paragon.Plugins.ScreenCapture/SnippingWindow.xaml.cs:72]: what is the significance of 80?

File: minuet/Paragon.Plugins.ScreenCapture/SnippingWindow.xaml.cs [hunk 3]
```diff
@@ -102,7 +107,7 @@ private void OnLoaded(object sender, RoutedEventArgs e)
 
         private void OnColorClick(object sender, RoutedEventArgs e)
         {
-            var menuItem = (MenuItem) sender;
+            var menuItem = (System.Windows.Controls.MenuItem) sender;
```
Comment (lneir) [inline minuet/Paragon.Plugins.ScreenCapture/SnippingWindow.xaml.cs:110]: why the change here and line 175 below?

File: minuet/Paragon.Plugins.ScreenCapture/SnippingWindow.xaml.cs (hunk 2 above)
Comment (rbarros-daitan) [reply minuet/Paragon.Plugins.ScreenCapture/SnippingWindow.xaml.cs:72]: Hi Lynn. This is what we have currently on code´s comments:
 // reposition the window so there's a neat effect of showing 
// the screenshot edit window in place of the selected region

File: minuet/Paragon.Plugins.ScreenCapture/SnippingTool.cs (hunk 1 above)
Comment (rbarros-daitan) [reply minuet/Paragon.Plugins.ScreenCapture/SnippingTool.cs:43]: StartPosition is a property of the form itself. Setting it to Manual is needed in order to have the form positioned on any of the available screens.

File: minuet/Paragon.Plugins.ScreenCapture/SnippingWindow.xaml.cs (hunk 3 above)
Comment (rbarros-daitan) [reply minuet/Paragon.Plugins.ScreenCapture/SnippingWindow.xaml.cs:110]: It was due to a conflict between MenuItem & ContextMenu existing on System.Windows.Controls & System.Windows.Forms.

**Comment by lneir** [comment]
lgtm
